Purchases/Networking/RCBackend.h

```diff
@@ -37,6 +37,12 @@ typedef void(^RCOfferSigningResponseHandler)(NSString * _Nullable signature,
                                              NSNumber * _Nullable timestamp,
                                              NSError * _Nullable error);
 
+typedef void(^PostRequestResponseHandler)(NSError * _Nullable error);
```

Comment:
if this is just an `Error -> Void`, do we even need the `typedef`? not a big deal, I just find them harder to read sometimes than the actual types if they're simple enough